FlatMate/Components/InputView.swift

```diff
@@ -11,27 +11,36 @@ struct InputView: View {
     @Binding var text: String
     let title: String
     let placeholder: String
-    var isSecureField = false
+    var isSecureField: Bool
     
     var body: some View {
         VStack(alignment: .leading) {
             Text(title)
                 .font(.custom("Outfit-ExtraLight", size: 17))
                 .padding(.bottom, -5)
             
-            TextField(placeholder, text: $text)
-                .font(.custom("Outfit-ExtraLight", size: 17))
-                .padding(.all, 12)
-                .background(Color("primaryBackground")) // Using the custom color
-                .cornerRadius(10) // Rounded corners
-
-                .foregroundColor(.white)
+            if isSecureField {
+                SecureField(placeholder, text: $text)
+                    .font(.custom("Outfit-ExtraLight", size: 17))
+                    .foregroundColor(.black) // This sets the color of the typed text
+                    .padding(.all, 12)
+                    .background(Color("primaryBackground")) // Using the custom color
+                    .cornerRadius(10) // Rounded corners
+            } else {
+                TextField(placeholder, text: $text)
+                    .font(.custom("Outfit-ExtraLight", size: 17))
+                    .foregroundColor(.black)
+                    .padding(.all, 12)
+                    .background(Color("primaryBackground")) // Using the custom color
+                    .cornerRadius(10) // Rounded corners
+            }
```

Comment:
Is it possible to apply the styles to both outcomes at once? Not a big deal but as-is this requires changing things twice if we want to modify the style.
Something like this (idk if the syntax is right)
```swift
{ isSecureField ? SecureField : TextField }(placeholder, text: $text).style
```